docs/release-notes/0.4/release-0.4.md

```diff
@@ -0,0 +1,82 @@
+# ML.NET 0.4 Release Notes
+
+Today we are releasing ML.NET 0.4. During this release we have started
+exploring new APIs for ML.NET that enable functionality that is missing from
+the current APIs. We welcome feedback and contributions to the
+[discussion](https://github.com/dotnet/machinelearning/projects/4). While the
+focus has been on designing the new APIs, we have also moved several
+components from the internal codebase to ML.NET.
+
+### Installation
+
+ML.NET supports Windows, MacOS, and Linux. See [supported OS versions of .NET
+Core
+2.0](https://github.com/dotnet/core/blob/master/release-notes/2.0/2.0-supported-os.md)
+for more details.
+
+You can install ML.NET NuGet from the CLI using:
+```
+dotnet add package Microsoft.ML
+```
+
+From package manager:
+```
+Install-Package Microsoft.ML
+```
```

Comment:
Should we link to how to build your own copy?

Reply:
I don't think this is necessary for the release notes (it's more about getting started with using the package as opposed to building it).